pkg/intent/intent.go

```diff
@@ -115,11 +116,26 @@ func (i *Store) SetPod(node string, manifest pods.PodManifest) (time.Duration, e
 	if err != nil {
 		return 0, err
 	}
+
+	podService := &consulapi.AgentServiceRegistration{
+		Name: manifest.ID(),
+	}
+	if manifest.StatusPort != 0 {
+		podService.Check = &consulapi.AgentServiceCheck{
+			// prints the HTTP response on stderr, while exiting 0 if the status code is 200
+			Script:   fmt.Sprintf(`if [[ $(curl 0.0.0.0:%v/_status -s -o /dev/stderr -w "%%{http_code}") == "200" ]] ; then exit 0 ; else exit 2; fi`, manifest.StatusPort),
```

Comment:
Sure it should be 0.0.0.0 ? Which machine runs the health check? I recall we wanted to assume the checks run on machines other than the app hosts.

Basically asking "should a VIP go here instead"